src/errors.rs

```diff
@@ -54,9 +56,10 @@ custom_error! {
     /// Supposed to be sent as numbers, though they are sent as string most of the time?
     ///
     /// Also includes errors when initiating a connection and unexpected opcodes
-    #[derive(PartialEq, Eq)]
+    #[derive(PartialEq, Eq, Default)]
```

Comment:
If this is `pub`, it should also have other standard derives like `Copy, Clone, Debug`